pbspro.spec

```diff
@@ -258,20 +258,9 @@ ${RPM_INSTALL_PREFIX:=%{pbs_prefix}}/libexec/pbs_postinstall client \
 %preun %{pbs_server}
 if [ "$1" != "1" ]; then
 	# This is an uninstall, not an upgrade.
-	if [ -x /sbin/chkconfig -a -x /sbin/service ]; then
-		out=`/sbin/chkconfig --list pbs 2>/dev/null`
-		if [ $? -eq 0 ]; then
-			if [ -x /etc/init.d/pbs ]; then
-				/etc/init.d/pbs stop
-			else
-				/sbin/service pbs stop
-			fi
-			/sbin/chkconfig --del pbs
-		fi
-	else
-		[ -x /etc/init.d/pbs ] && /etc/init.d/pbs stop
-		rm -f /etc/rc.d/rc?.d/[KS]??pbs
-	fi
+	[ -x /etc/init.d/pbs ] && /etc/init.d/pbs stop
+	[ -x /sbin/chkconfig ] && /sbin/chkconfig --del pbs
```

Comment:
Earlier we were checking if pbs is a registered service, Would it throw an error if pbs is not a registered service? Or, is it never the case that pbs is not a registered service?